ERCS/erc-7204.md

```diff
@@ -0,0 +1,210 @@
+---
+eip: 7204
+title: Contract wallet management token
+description: Designed for simple token
+author: Xiang (@wenzhenxiang), Ben77 (@ben2077), Mingshi S. (@newnewsms)
+discussions-to: https://ethereum-magicians.org/t/token-asset-management-interface-with-smart-contract-wallet/14759
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-21
+requires: 165
+---
+
+## Abstract
+
+A proposal to manage simple tokens by the user's smart contract wallet, which provides a new way to manage assets, utilizes the programmability of the smart contract wallet, and also provides more playability.
```

Comment:
Your abstract should give a high-level technical overview of your proposal. It shouldn't just be an introduction. Readers should be able to get a sense of how the proposal works, just from this section.